applications/smart-distancing/ui/web_gui.py

```diff
@@ -74,16 +82,28 @@ def update(self, input_frame, nn_out, distances):
         # Put fps to the frame
         # region
         # -_- -_- -_- -_- -_- -_- -_- -_- -_- -_- -_- -_- -_- -_-
-        txt = 'Frames rate = ' + str(self._displayed_items['fps']) + '(fps)'  # Frames rate = 95 (fps)
+        txt_fps = 'Frames rate = ' + str(self._displayed_items['fps']) + '(fps)'  # Frames rate = 95 (fps)
         # (0, 0) is the top-left (x,y)
-        origin = (10, 470)
-        vis_util.text_putter(input_frame, txt, origin)
+        origin = (10, 450)
```

Comment:
would you please change this hard code values with values that is relative to resolution parameter in config file?